OmgvaPOS/src/PaymentManagement/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,113 @@
+using Microsoft.AspNetCore.Mvc;
+using OmgvaPOS.Database.Context;
+using OmgvaPOS.HelperUtils;
+using OmgvaPOS.PaymentManagement.DTOs;
+using OmgvaPOS.PaymentManagement.Models;
+
+using Stripe;
+using PaymentMethod = OmgvaPOS.PaymentManagement.Enums.PaymentMethod;
+
+namespace OMGVA_PoS.Business_layer.Controllers
+{
+    [ApiController]
+    [Route("payment")]
+    public class PaymentController : ControllerBase
+    {
+        private readonly OmgvaDbContext _context;
+        public PaymentController(OmgvaDbContext context)
+        {
+            _context = context;
+        }
+
+        [HttpPost]
+        [Route("process-card")]
+        public IActionResult ProcessCardPayment([FromBody] PaymentRequest request)
+        {
+            var businessId = JwtTokenHandler.GetTokenBusinessId(HttpContext.Request.Headers.Authorization);
+            if (businessId == null)
+            {
+                return Unauthorized(new { error = "Unauthorized" });
```

Comment:
This should be Forbid. But maybe Mantas will fix it. Feel free to resolve, but keep in mind that we use forbid when authenticated user lacks authorisation.